README.md

```diff
@@ -275,7 +312,37 @@ async function sign(asset, manifest) {
   });
 }
 
-sign(asset, manifest);
+// build a manifest to use for signing
+const manifest = new ManifestBuilder(
```

Comment:
A bit repetitive, but it makes the examples copy-pastable to have the manifest buildet here too